server/Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'mongoid', '~> 4.0.0'
 gem 'symmetric-encryption'
 gem 'activesupport', '~> 4.1.14'
 gem 'faye-websocket', '~> 0.10.0'
+gem 'eventmachine', '>= 0.12.0'
```

Comment:
Server has been using eventmachine from the beginning.. I just thought that 0.12.0 version is somewhat misleading here.